README.md

```diff
@@ -1,6 +1,7 @@
 # Standardized Debug
 
-This proposal is at Stage 1 of the [TC39 Process](https://tc39.es/process-document/).
+This proposal is at Stage 1 of the [TC39 Process](https://tc39.es/process-document/).<br>
```

Comment:
```suggestion
This proposal is at Stage 1 of the [TC39 Process](https://tc39.es/process-document/).
```
it's markdown, explicit line breaks aren't required.

Reply:
```suggestion
This proposal is at Stage 1 of the [TC39 Process](https://tc39.es/process-document/).

```

Reply:
thx, markdown actually needs 2 line-breaks to initiate a newline.